recipes/jfalcou-eve/all/conanfile.py

```diff
@@ -1,6 +1,6 @@
-from conan import ConanFile, tools
-from conan.tools.scm import Version
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.tools import files, build
```

Comment:
These are not the recommend https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#python-import-statements

You will need to go one level deeper 🙏

Reply:
Importing `form conan.tools import xxx` is ok. The linter is already validating it according to documented https://docs.conan.io/en/latest/reference/conanfile/tools.html